pages/_app.js

```diff
@@ -9,6 +9,8 @@ import {
   injectTooltipStyles,
 } from "~/common/styles/global";
 
+import "./css/prism.css";
```

Comment:
Lets follow the other pattern here. Instead of the import of "./css/prism.css", lets create a new `injectCodeBlockStyles`